src/store/modules/general.store.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+export default {
+	state: () => ({
+		isLoading: true
+	}),
+	mutations: {
+		loading(state, payload) {
+			state.isLoading = payload;
+		}
+	},
+	actions: {
+		changeLoading({ commit }) {
+			setTimeout(() => {
+				commit("loading", false);
+			}, 3000);
+		},
+		async getPaginationPayload(context, category) {
```

Comment:
the name suggests that the only kind of data that we can retrieve here is the payload number itself. Is this really the case here?

Reply:
Really ? For me it is payload in pagination mode

Reply:
then how about `getPayloadInPaginationMode`?